pkg/apis/toolchain/v1alpha1/usersignup_types.go

```diff
@@ -10,6 +10,9 @@ const (
 	UserSignupApproved ConditionType = "Approved"
 	// UserSignupComplete means provisioning is complete
 	UserSignupComplete ConditionType = "Complete"
+
+	// UserSignupUserEmailLabelKey is used for the usersignup email annotations key
+	UserSignupUserEmailLabelKey = "toolchain.dev.openshift.com/user-email"
```

Comment:
It's not a label. It's annotation Key.
We can't put email to a label. It's not DNS compliant. We have to use annotations.

```suggestion
	// UserSignupUserEmailAnnotationKey is used for the usersignup email annotations key
	UserSignupUserEmailAnnotationKey = "toolchain.dev.openshift.com/user-email"
```

Reply:
@alexeykazakov sorry, i changed this PR to WIP because @xcoulon had suggested i change it to label instead of annotation..

Reply:
We can't use labels to keep email. Emails are not DNS compliant. We have to use annotations.

Reply:
fixed: 660995a069185df07cbd54a27d4f66ea3c2c5264